package.json

```diff
@@ -121,6 +121,7 @@
     "redux-thunk": "1.0.0",
     "rtlcss": "2.0.5",
     "sanitize-html": "1.11.1",
+    "scrollbar-width": "3.1.1",
```

Comment:
We will need to update the [shrinkwrap](https://wpcalypso.wordpress.com/devdocs/docs/shrinkwrap.md), too.